packages/gitmoji-changelog-markdown/src/template.md

```diff
@@ -9,6 +9,14 @@
 
 {{#each commits}}
 - {{emoji}} {{message}} ({{hash}})
+  {{#if siblings}}
+
+    **siblings:**
```

Comment:
I think we could remove the **siblings:** word and just keep the bullet hierarchy